openeo/extra/spectral_indices/spectral_indices.py

```diff
@@ -0,0 +1,99 @@
+from openeo.processes import ProcessBuilder, array_modify
+from openeo.rest.datacube import DataCube
+import numpy as np
+
+def _get_expression_map(cube: DataCube, x: ProcessBuilder):
+    collection_id = cube.graph["loadcollection1"]["arguments"]["id"]
+    bands = [band.replace("0", "").upper() for band in cube.metadata.band_names]
+
+    def check_validity():
+        if not all(band in orig_bands for band in bands):
+            raise ValueError(
+                "The bands in your cube {} are not all Sentinel-2 bands (the following are: {})".format(bands,orig_bands))
```

Comment:
error mentions "Sentinel-2" but if I understand correctly there is also support for Landsat, probaV, Modis, ...